Sources/App/Models/Package.swift

```diff
@@ -111,6 +111,7 @@ extension Package {
         case analysisFailed = "analysis_failed"
         case ingestionFailed = "ingestion_failed"
         case invalidCachePath = "invalid_cache_path"
+        case unexistentCacheDir = "unexistent_cache_dir"
```

Comment:
I should have tried with the closest word to Spanish (`inexistente` 😉) instead of inventing words 😂. I like the longer, but clearer name `cacheDirectoryDoesNotExist `